src/main/java/org/apache/mesos/specification/TaskTypeSpecification.java

```diff
@@ -1,8 +1,16 @@
 package org.apache.mesos.specification;
 
+import org.apache.mesos.Protos;
+
+import java.util.Collection;
+
 /**
- * A TaskTypeSpecification describes the count of a particular TaskSpecification which should be launched.
+ * A TaskTypeSpecification describes a Type of Task and how many should be launched.
  */
-public interface TaskTypeSpecification extends TaskSpecification {
+public interface TaskTypeSpecification {
```

Comment:
This looks like you don't define a command at the `TaskType` layer.  Is that the intention?